src/PayU/PaymentsApi/AluV3/Services/ResponseBuilder.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+
+namespace PayU\PaymentsApi\AluV3\Services;
+
+use PayU\Alu\Exceptions\ClientException;
+use PayU\Alu\Response;
+use PayU\Alu\ResponseWireAccount;
+use PayU\PaymentsApi\AluV3\Entities\AuthorizationResponse;
+use PayU\PaymentsApi\AluV3\Exceptions\ResponseBuilderException;
+use SimpleXMLElement;
+
+final class ResponseBuilder
```

Comment:
Maybe in a similar way to this class we could create a new responsebuilder for 3ds response, in order to extract the code from\PayU\Alu\Client::getThreeDSReturnResponse which returns a PayU\Alu\Response as well.

Reply:
This will be made in a further PR, the aim for this one was to refactor the non 3ds authrization.